models/sessions/scratch/snowflake/snowplow_web_sessions_this_run.sql

```diff
@@ -181,6 +215,17 @@ select
     -- engagement fields
     c.page_views,
     c.engaged_time_in_s,
+    {%- if var('snowplow__list_event_counts', false) %}
+    TRY_PARSE_JSON(c.event_counts_string) as event_counts,
```

Comment:
Should we have an `else` here so this is always populated in case they want to enable it later?

Reply:
If they enable it later it will just add the column to the end of their table and populate from then onwards